mimic/model/lstm_model.py

```diff
@@ -32,15 +32,16 @@ def learn(self, text):
         """Use input text to train the LSTM model."""
         # Clean & verify text
         logging.info('Cleaning and verifying text')
-
+        text = text[:150000]
```

Comment:
Not a huge deal, but it would be nice to have a comment here explaining why `150000`/what that choice means in terms of the learning process.